exploration/error-handling.md

```diff
@@ -87,6 +99,26 @@ For example, in ICU,
 [the suggested practice](https://docs.google.com/document/d/11yJUWedBIpmq-YNSqqDfgUxcREmlvV0NskYganXkQHA/edit#bookmark=id.lx4ls9eelh99)
 is to avoid additionally returning optional error codes when providing best-effort formatted results.
 
+### Require a best-effort message value and signaling of an error
+
+> In all cases, when encountering an error,
+> a message formatter MUST be able to signal an error or errors.
+> It MUST also provide the appropriate fallback representation of the _message_ defined
+> in this specification.
+
+This alternative maintains that an implementation must both return a best-effort message
+and must signal that an error has occurred.
+Compared to the current text,
+this alternative slightly loosens the constraint of 
+returning an error into only needing to signal an error.
```

Comment:
I think this is slightly misleading, in that it would be conformant to have two `format` methods, one that returns an error and one that returns the fallback message (cf. the conversation with @eemeli) 

Perhaps:

```suggestion
This alternative requires that an implementation provide both an error signal
and a means of accessing a "best-effort" fallback message.
This slightly relaxes the requirement of "returning" an error
(to allow a locally-appropriate signal of the error).

Under this alternative, implementations can be conformant by providing
two separate formatting methods or functions,
one of which returns the fallback string and one of which signals the error.
```